.github/workflows/node.js.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 #     runs-on: ubuntu-latest
 #     steps:
 #       - uses: actions/checkout@v1
-#       - uses: actions/setup-node@v1
+#       - uses: actions/setup-node@v1.1.1
```

Comment:
@dependabot Comments should not be considered as valid versions, right?

Reply:
Ah thanks for flagging this! @greysteil has been working on GitHub actions support but is away until next week.

Reply:
Yeah, you probably do want this change though, no? Only got picked up because there was an uncommented declaration for the same dependency.

Reply:
> Only got picked up because there was an uncommented declaration for the same dependency.

Good to know. 👌

> you probably do want this change though

Well, here it's OK'ish, but it feels very weird. 

I'm concerned about situations like the following:

```yaml
        - uses: actions/setup-node-other@anyway
#       - uses: actions/setup-node@v1
```

Reply:
The way we do updating is robust enough to avoid that, don't worry 🙂